### PR TITLE
Add voxel_min_points param for voxel filter

### PR DESCRIPTION
**Pull request by akiroz**
This feature prevents "ghost points" from noise generated by depth cameras.

### CODE REVIEW

**Comment by SteveMacenski** [comment]
LGTM, please also make that update to the `eloquent-devel` branch as well so its reflected in ROS2

**Comment by akiroz** [comment]
@SteveMacenski This PR is ready for review along with #161

File: README.md [hunk 1]
```diff
@@ -116,6 +116,7 @@ rgbd_obstacle_layer:
     inf_is_valid: false          #default false, for laser scans
     clear_after_reading: true    #default false, clear the buffer after the layer gets readings from it
     voxel_filter: true           #default off, apply voxel filter to sensor, recommend on 
+    voxel_min_points: 10         #default 0, minimum points per voxel for voxel filter
```
Comment (SteveMacenski) [inline README.md]: Please change this to `0`.

File: src/measurement_buffer.cpp [hunk 2]
```diff
@@ -160,6 +161,7 @@ void MeasurementBuffer::BufferROSCloud(const sensor_msgs::PointCloud2& cloud)
       sor.setLeafSize ((float)_voxel_size,
                        (float)_voxel_size,
                        (float)_voxel_size);
+      sor.setMinimumPointsNumberPerVoxel((unsigned int) _voxel_min_points);
```
Comment (SteveMacenski) [inline src/measurement_buffer.cpp]: please use `static_cast<unsigned int>(_voxel_min_points);`

**Comment by akiroz** [comment]
Both PRs updated, thanks!

**Comment by SteveMacenski** [comment]
Thanks for your contribution!

**Pull request merged**